modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-attaching-a-resource-object-to-your-query_{context}"]
+= Attaching a resource object to your query
+
+This procedure explains how to attach a resource object to provided additional context for your chat query. 
```

Comment:
This procedure explains how to attach a resource object to provide additional context for your query.

Reply:
done